packages/optimistic-oracle/test/index.js

```diff
@@ -43,10 +64,9 @@ contract("index.js", function() {
       assert.notStrictEqual(spyLogLevel(spy, i), "error");
     }
 
-    // Logger should emit two logs: one indicating the start of the runner, and one about the early termination of the
-    // of the loop because the pollingDelay is set to 0.
-    assert.strictEqual(spy.getCalls().length, 2);
+    // The first log should indicate that the OO-Keeper runner started successfully
+    // and auto detected the OO's deployed address.
     assert.isTrue(spyLogIncludes(spy, 0, "OO keeper started"));
-    assert.isTrue(spyLogIncludes(spy, 1, "End of serverless execution loop - terminating process"));
+    assert.isTrue(spyLogIncludes(spy, 0, optimisticOracle.address));
```

Comment:
why did you make this change? Surely you still want to double-check it exits at the end of the loop?

Reply:
Yes good point I'll change

Reply:
Done